.github/workflows/gradle.yml

```diff
@@ -65,6 +65,12 @@ jobs:
           check_name: Java CI / Test Report (${{ matrix.java }})
           report_paths: '**/build/test-results/test/TEST-*.xml'
           check_retries: 'true'
+      - name: "📜 Upload binary compatibility check results"
+        if: always()
```

Comment:
Is the report always generated?

Reply:
If you run `check` it should, yes. the `always()` is here in case build fails, we still need to upload the report.